selfdrive/car/ford/fordcan.py

```diff
@@ -128,17 +128,19 @@ def create_acc_msg(packer, CAN: CanBus, long_active: bool, gas: float, accel: fl
   Frequency is 50Hz.
   """
 
-  decel = accel < 0 and long_active
+  actuateBrakes = gas == -5 and long_active
```

Comment:
This has had the most significant improvement to Long control. I've yet to hear of anyone with a Q3 harness trying this.